### ISSUE TITLE
[Feature testing] clasp swallows or delays output to the terminal

### ISSUE THREAD

**Issue by oshliaer** [PopGoesTheWza/clasp#28]
## Intro

I'm from there [google/clasp/issues/805](https://github.com/google/clasp/issues/805).

My problem is that the clasp is swallowing the output or locking the output to the terminal.

I'm using spawn calls to run clasp under the webpack watch mode.

The main approach is

```js
spawn('clasp', 'run', {
  stdio: 'inherit',
});
```

Two things happen there.

  1. сlasp pushes data. But it does not add output to console
  2. the console doesn't respond to Ctrl+C input. I need to call an event from webpack (it is called first in the current console) then I can press CTRL+C.

This rarely happens. I don't find a pattern.

## The reasons for this topic

I'm going to test this fork for the same problem. I will respond within 2 weeks.

**Comment by oshliaer** [comment]
While testing I did not find any problems with terminal delays. But I still don't understand why it is happening with origin clasp :man_shrugging:.

**Comment by PopGoesTheWza** [comment]
@contributorpw original repo no longer has google maintainers. There is no one to release an updated `@google/clasp` npm package currently.

This forked repo contains all the refactored code I merged into the official repo (but can't release on npm) and a few additions.

I released and update a `forked-clasp` npm package to make the many fixes and new features available.

Since the last official `google/clasp` npm release, I did a massive refactor of the code, especially to cleanup pending promises and loose async function which are likely related to your original issue.
